packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -164,24 +164,28 @@ const PersonalDetails = ({
                                         style={{ paddingBottom: isDesktop() ? 'unset' : null }}
                                     >
                                         {!is_dashboard && <FormSubHeader title={localize('Title and name')} />}
-                                        {'salutation' in props.value && ( // TODO: [deriv-eu] Remove salutation once api is optional
```

Comment:
why are we removing this condition?

Reply:
Sorry I made it by mistake.. Was testing someting. Thanks for attention